felix/rules/static.go

```diff
@@ -998,39 +998,94 @@ func (r *DefaultRuleRenderer) StaticRawTableChains(ipVersion uint8) []*Chain {
 	}
 }
 
-func (r *DefaultRuleRenderer) StaticBPFModeRawChains(ipVersion uint8, tcBypassMark uint32) []*Chain {
-	rawPreroutingChain := &Chain{
-		Name: ChainRawPrerouting,
-		Rules: []Rule{
-			Rule{
-				// Return, i.e. no-op, if bypass mark is not set.
-				Match:  Match().NotMarkMatchesWithMask(tcBypassMark, 0xffffffff),
-				Action: ReturnAction{},
-			},
-			// At this point we know bypass mark is set, which means that the packet has
-			// been explicitly allowed by untracked ingress policy (XDP).  We should
-			// clear the mark so as not to affect any FROM_HOST processing.  (There
-			// shouldn't be any FROM_HOST processing, because untracked policy is only
-			// intended for traffic to/from the host.  But if the traffic is in fact
-			// forwarded and goes to or through another endpoint, it's better to enforce
-			// that endpoint's policy than to accidentally skip it because of the BYPASS
-			// mark.  Note that we can clear the mark without stomping on anyone else's
-			// logic because no one else's iptables should have had a chance to execute
-			// yet.
-			Rule{
-				Action: SetMarkAction{Mark: 0},
-			},
-			// Now ensure that the packet is not tracked.
-			Rule{
-				Action: NoTrackAction{},
-			},
+func (r *DefaultRuleRenderer) StaticBPFModeRawChains(ipVersion uint8, wgEncryptHost, enforceRPF bool) []*Chain {
+
+	var rawRules, rpfRules []Rule
+
+	rpfChainName := ChainNamePrefix + "RPF"
```

Comment:
Suggest making a constant alongside the other names in rules_def.go. 

Nit: I'd use lower case `rpf`, the convention was meant to be that upper case was used only for top-level chains (to match the iptables convention).  Doesn't really matter, but that was the intention.